client/js/components/procedure/admin/DpNewProcedure/DpNewProcedure.vue

```diff
@@ -7,53 +7,288 @@
+          id="r_name"
+          :label="{ text: Translator.trans('name') }"
+          :maxlength="200"
+          name="r_name"
+          required
+          type="text" />
+      </dp-form-row>
+      <!--      {% block master %}-->
+      <dp-form-row v-if="hasPermission('feature_procedure_templates')">
+        <dp-select
+          id="blueprint"
+          :label="{
+            hint: blueprintHint,
+            text: Translator.trans('master')
+          }"
+          name="r_copymaster"
+          :options="blueprintOptions"
+          :selected="masterBlueprintId"
+          @select="setBlueprintData" />
+      </dp-form-row>
+
+<!--      {# Only show select if there is more than one choice. Otherwise, pass the id as the value of a hidden field. #}-->
+      <div v-if="procedureTypes.length > 1">
+        <dp-label
+          for="r_procedure_type"
+          :hint="Translator.trans('text.procedures.types.hint')"
+          :text="Translator.trans('text.procedures.type')"
+          required />
+        <dp-multiselect
+          v-model="currentProcedureType"
+          class="layout__item u-1-of-1 u-pl-0 u-mb display--inline-block"
+          label="name"
+          :options="procedureTypes"
+          required
+          track-by="id">
+          <template v-slot:option="props">
+            {{ props.option.name }}<br>
+            <span class="font-size-small">{{ props.option.description }}</span>
+          </template>
+        </dp-multiselect>
+        <input
+          type="hidden"
+          name="r_procedure_type"
+          :value="currentProcedureTypeId">
+      </div>
+<!--      {# There should always be at least one procedureType defined #}-->
+      <input
+        v-else
+        name="r_procedure_type"
+        type="hidden"
+        :value="procedureTypes[0].id">
+
+      <dp-form-row>
+        <dp-input
+          id="main-email"
+          data-cy="agencyMainEmailAddress"
+          :label="{
+            hint: Translator.trans('explanation.organisation.email.procedure.agency'),
+            text: Translator.trans('email.procedure.agency')
+          }"
+          name="agencyMainEmailAddress[fullAddress]"
+          required
+          type="email"
+          :value="mainEmail" />
+      </dp-form-row>
+      <dp-form-row>
+        <dp-text-area
+          v-bind="{
+            ...internalNote
+          }" />
+      </dp-form-row>
+
+      <div class="u-mb-0_75">
+        <dp-label
+          for="startdate"
+          :hint="periodHint"
+          :required="hasPermission('feature_auto_switch_to_procedure_end_phase')"
+          :text="Translator.trans('period')" />
+
+        <dp-date-range-picker
+          class="u-2-of-4"
+          start-id="startdate"
+          start-name="r_startdate"
+          end-id="enddate"
+          end-name="r_enddate"
+          :required="hasPermission('feature_auto_switch_to_procedure_end_phase')"
+          :calendars-after="2"
+          :enforce-plausible-dates="true">
```

Comment:
If You just want to pass true, You can simplify it like so:
```suggestion
          enforce-plausible-dates>
```